src/views/Settings.vue

```diff
@@ -407,6 +437,29 @@ export default defineComponent({
     const store = useStore();
     const router = useRouter();
 
+    const productIdentificationStore = useProductIdentificationStore();
+    const productIdentificationOptions = productIdentificationStore.getProductIdentificationOptions;
+
+    // Injecting identifier preference from app.view
+    const productIdentificationPref: any  = inject("productIdentificationPref");
+
+    // Function to set the value of productIdentificationPref using dxp-component
+    const setProductIdentificationPref = (value: string, id: string) =>  {
+      const eComStore = store.getters['user/getCurrentEComStore']; 
+
+      // If productPreference value is same as ion change value then not calling the set function as there is no cahnge 
+      if(eComStore.productStoreId && (productIdentificationPref.value[id] !== value)){
+        productIdentificationStore.setProductIdentificationPref(id, value, eComStore.productStoreId)
+          .then(() => {
+            showToast(translate("Product identifier preference updated"));
+          })
+          .catch((error) => {
+            showToast(translate("Failed to update Product identifier preference"));
```

Comment:
```suggestion
            showToast(translate("Failed to update product identifier preference"));
```
Also add the entry in json filed